tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "moduleResolution": "node",
     "esModuleInterop": false,
     "noEmitOnError": true,
-    "lib": ["esnext", "dom", "dom.iterable"],
+    "lib": ["esnext", "dom", "dom.iterable", "webworker"],
```

Comment:
This is tricky—we want the `webworker` library to be available in the service worker, but not in other code. Conversely, the `dom` and `dom.iterable` libraries should not be available in the service worker. Jake has a repository w/ the following example, where includes are used in `tsconfig.json` to enable different environments:

https://github.com/jakearchibald/typescript-worker-example

Once the commands work, we can codify them in `package.json`.